src/main/java/com/hotels/bdp/circustrain/bigquery/partition/HivePartitionKeyAdder.java

```diff
@@ -40,13 +39,13 @@ public Table add(String partitionKey, Schema filteredTableSchema) {
 
     Table newTable = new Table(table);
 
-    BigQueryToHiveTypeConverter typeConverter = new BigQueryToHiveTypeConverter();
     for (Field field : filteredTableSchema.getFields()) {
       String fieldName = field.getName().toLowerCase().trim();
       if (partitionKey.equals(fieldName)) {
         FieldSchema fieldSchema = new FieldSchema();
         fieldSchema.setName(fieldName);
-        fieldSchema.setType(typeConverter.convert(field.getType().toString()).toLowerCase());
+        // all partition types are string
+        fieldSchema.setType(FIELD_SCHEMA_TYPE);
```

Comment:
I guess this is also a change that should be mentioned in the changelog?

Reply:
Done 👍